README.md

```diff
@@ -37,6 +37,10 @@
 
 > #### Click [here](http://codeastrike.000webhostapp.com/hbd/) to view live demo!
 
+### Designing another [website](https://codepen.io/randomname-akh/pen/gOOOzPB) to celebrate your loved one's day!
+
```

Comment:
@akhillllldev This pen is really nice, can you please place the code inside `examples/` folder?

Reply:
@vinitshahdeo Sure.